### ISSUE TITLE
VPAT — <meta> Tag Errors

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#5000]
As noted in an internal accessibility audit, there are errors with a `<meta>` tag.  This particular tag should not have a `name` attribute, but should have either an `itemprop` or `property` attribute.

Note that `itemprop` requires the element to be a property of an item.  q.v. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemprop

(I recommend using `property` for now)

![Screen Shot 2020-01-31 at 8 47 06 PM](https://user-images.githubusercontent.com/12703624/73585198-d3df3980-446c-11ea-85f8-e60fad6f61c0.png)

**Comment by elihschiff** [comment]
I am a bit confused on why it should not have a name attribute. I tried to look this up online and cannot fine anything. Do you happen to have a link on where I can learn more about this topic?

From my understanding, having `<meta name="viewport" content="..." />` is the normal and expected way for a website to be made to work well on mobile screens. This is recommended by Mozilla [here](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag), Google [here](https://developers.google.com/web/tools/lighthouse/audits/has-viewport-meta-tag), and Bootstrap (which we use) [here](https://getbootstrap.com/docs/4.4/getting-started/introduction/#responsive-meta-tag). Also I did try and remove it to see what would happen and the website no longer worked on mobile.

**Comment by pbailie** [comment]
I ran the code through W3C's HTML validator (https://validator.w3.org/) and received the following in its report:

9. Error: Attribute `name` not allowed on element `meta` at this point.
10. Error: Element `meta` is missing one or more of the following attributes: `itemprop`, `property`.

![Screen Shot 2020-02-03 at 2 00 54 PM](https://user-images.githubusercontent.com/12703624/73682562-9a861400-468e-11ea-801a-983b56a36077.png)

We should be very careful with proper HTML adherence for assistive technologies.  The other errors in the screenshot are very minor and easy to correct.  Separate issues can be opened as needed.

Also as a reminder: web accessibility is Rensselaer campus policy, so we cannot ignore it because there is a conflict with mobile functionality.  Especially since the desktop site is predominantly used by students for coursework.  Although satisfying both would be great.

**Comment by elihschiff** [comment]
Thanks for linking that website. I think I found the root cause of the issue. The issue seems to be that the meta tag is in the wrong place, it should be inside the head tag, not the body. The website sees a meta tag inside the body and therefore expects different values from a meta in the head.

When I tested a version of the page with the meta tag in the head that fixed the issue (and the mobile website still works).

I will look into fixing this.

**Comment by elihschiff** [comment]
As a followup, is the goal to fix every "error" across all of Submitty, or will you be making issues here on github for specific ones that need to fixed (just like this issue)?

**Comment by pbailie** [comment]
@elihschiff 
I will be making specific issues on Github so they are documented.  I am working on a few different things right now, so you may see new issues come in as periodic bunches rather than as a steady flow.
